brainstormforce/ultimate-addons-for-gutenberg
# refactor: FAQ Block's HTML, Classes & CSS.

### Description
Report - https://docs.google.com/document/d/1ZNsQoG4eEtac9_bxWwwt6XBkOSdfarQ3D9vaulpcss0/edit#

### Types of changes
 Bug fix (non-breaking change which fixes an issue) 

### How has this been tested?
 - CHeck Backward compatibility.
 - Check each & every setting.

### Checklist:
